config/electronBuilder.config.js

```diff
@@ -50,6 +59,7 @@ module.exports = {
                 ? s3Bucket[process.env.ELECTRON_ENV]
                 : 'ad-hoc-expensify-cash',
             channel: 'latest',
+            path: isCorrectElectronEnv ? s3Path[process.env.ELECTRON_ENV] : '/',
```

Comment:
I wonder if we should just fail the Electron build early if `process.env.ELECTRON_ENV` is not set?

Reply:
Yes, you may be right, if there isn't any `process.env.ELECTRON_ENV` it means that the command is not correct. I've removed those checks from the `electronBuilder.config.js` file